operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -462,6 +467,10 @@ protected void addWarningsToStatus(Status status, Set<Condition> unknownAndDepre
         return selector;
     }
 
+    public Map<String, String> getMatchLables() {
+        return selector.get().getMatchLabels();
+    }
```

Comment:
Is this really needed? It does not seem to be used in any production code.

Reply:
Actually I created this to use in the test. In the Operator metrics test, when Iam asserting to check if the labels is correct, I need this method

Reply:
I do not think you need this. In the tests:
* You set the selector, so you need its value
* In this PR, you do not care about the selector value really. You care about the selector tag on the metric. That is what you need to test.

In case it would really be needed, it should probably:
* Not be public
* Have a Javadoc explaining it why and where it is needed